waku/waku.md

```diff
@@ -326,6 +332,16 @@ The supported codes:
 
 The drawback of sending message confirmations is that it increases the noise in the network because for each sent message, a corresponding confirmation is broadcasted by one or more peers.
 
+
+#### P2P Request
+
+This packet is used for sending Dapp-level peer-to-peer requests, e.g. Waku Mail Client requesting old messages from the Waku Mail Server.
```

Comment:
What do we mean by Dapp-level here?

Also please link to mailserver spec

Reply:
Not sure, this section has only been moved @adambabik